pkg/render/logstorage.go

```diff
@@ -57,7 +57,7 @@ const (
 
 	ElasticsearchNamespace = "tigera-elasticsearch"
 
-	TigeraElasticsearchCertSecret         = "tigera-secure-elasticsearch-cert"
```

Comment:
I don't think we should rename this to include Gateway. It is true this secret is used by the ESGateway but it is created as the ES secret that will be externally visible.
If we want to change the name I'd say something like ExternalESCertSecretName. If we want another const for the ESGatewayCertSecret then I'd say we create a 2nd with the same value.

Reply:
Const name restored to `TigeraElasticsearchCertSecret`